content/output/demographic/_index.Rmd

```diff
@@ -0,0 +1,98 @@
+---
+title: "Create Demographic table"
+author: "Tadeusz Lewandowski"
+date: "16/08/2022"
+output: html_document
+description: "An example of how to generate an Demogrphic output based on ADaM dataset"
+---
+
+## Introduction
+
+This guide will show you how two pharmaverse packages, along with some from tidyverse, can be used to create a Demographic table as an end-to-end, using the Admiral ADSL data as input.
+
+The two packages used with a brief description of their purpose are as follows:
+
+* [`{rtables}`](https://roche.github.io/rtables/index.html): designed to create and display complex tables with R.
+* [`{tern}`](https://insightsengineering.github.io/tern/main/index.html): contains analysis functions to create tables and graphs used for clinical trial reporting.
+
+It is important to understand [`{rtables}`](https://roche.github.io/rtables/index.html) grammar by reading through the above linked package site, as these are fundamental to being able to use [`{tern}`](https://insightsengineering.github.io/tern/main/index.html). 
```

Comment:
Would we be able to add a section on how to bring this display into an IDD module via teal or would you say best to do that as a separate example?

Reply:
I will have a separate example for APPs